content/pages/download.md

```diff
@@ -100,6 +100,8 @@ versions:
         packages:
           - slug: win64
             name: 64-Bit
+          - slug: winarm
+            name: ARM
```

Comment:
```suggestion
          - slug: win64
            name: x64
          - slug: winarm
            name: ARM64
```

Reply:
Windows supports the following processor architectures for executables:
![grafik](https://github.com/user-attachments/assets/dd82d6c3-1192-42be-a6cf-125a97adca9c)
Our executables are for `Arm64` not for `Arm`. I suggest to adjust the case to `x64` and `ARM64`.